### PR TITLE
Added two Chrony plugins and example graphs

### PR DESCRIPTION
**Pull request by Kolkman**
Added chrony_ , a wildcard plugin to track sourcestats per indiviual time source
Added chrony_sourcestats, a plugin that displays the sourcestats characteristics for several timesources in one graph.
Added example graphs for both plugins.

### CODE REVIEW

File: plugins/chrony/chrony_ [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+	}else{
+		Data::Validate::IP->import();
+	}
+
+
+}
+
+
+
+my $chronyc = $ENV{'chrony'} || "/usr/local/bin/chronyc";
```
Comment (sumpfralle) [inline plugins/chrony/chrony_:105]: How about the following instead?

```perl
my $chronyc = $ENV{'chrony'} || `which chronyc`;
```

File: plugins/chrony/chrony_sourcestats [hunk 2]
```diff
@@ -0,0 +1,375 @@
+#!/usr/bin/perl -w
+# -*- mode: cperl; cperl-indent-level: 8; -*-
+
+=head1 NAME
+
+chrony_sourcestatss - Plugin to monitor Chrony offsets for various hosts.
```
Comment (sumpfralle) [inline plugins/chrony/chrony_sourcestats:6]: Remove one `s` from `chrony_sourcestatss`?

**Review by sumpfralle — COMMENTED** [review]
Thank you for your contribution!

Please take a look at my comments below.

Additionally you should rename the example graphs in order to visualize them in the [munin plugin gallery](https://gallery.munin-monitoring.org/):
* `chrony_sourcestat_freq.png` lacks an `s` for `sourcestats`
* all graphs: use the suffix `day`, `week`, `month`, `year` or a number
    * all other suffixes are ignored

Thank you!

**Comment by Kolkman** [comment]
fixed

**Comment by sumpfralle** [comment]
Thank you for your fixes!

Regarding the names of the example graphs: in order to allow a reliable mapping from example graphs to plugins, the names really need to use the template `${PLUGIN_NAME}-${SUFFIX}`, with `SUFFIX` being one of `day`, `week`, `month`, `year` or a number. Thus you need to remove the example domain names and the descriptive suffixes.
Thank you!

**Comment by Kolkman** [comment]
>, the names really need to use the template `${PLUGIN_NAME}-${SUFFIX}`, with `SUFFIX` being one of `day`, `week`, `month`, `year` or a number. 


So, just to be sure, for the wildcard plugin 'chrony_' the example graph should be named 'chrony_-day' (i.e. with underscore and a dash)?

**Comment by sumpfralle** [comment]
> So, just to be sure, for the wildcard plugin 'chrony_' the example graph should be named 'chrony_-day' (i.e. with underscore and a dash)?

yes, exactly.

The relevant documentation is hidden in the [Munin Guide](http://guide.munin-monitoring.org/en/latest/develop/plugins/plugin-gallery.html?highlight=gallery#rules-for-plugin-contributors).

The corresponding plugin gallery implementation detail is [here](https://github.com/munin-monitoring/munin-plugin-gallery/blob/97a5845dc61b8f598b2e4675e49e873f6d583c40/plugin-gallery-generator#L214).

Thank you!

**Comment by Kolkman** [comment]
Thank you Lars, renamed the examples.

--Olaf

**Comment by sumpfralle** [comment]
Thank you!

**Pull request merged**